### DISCUSSION TITLE
来点邦布瑟图

### DISCUSSION THREAD

**Discussion by jenemi** [share121/inter-knot#510] [其他]
![image](https://github.com/user-attachments/assets/58955cad-6056-4106-bedc-dafc91ff4d1d)

**Comment by xinxingjie** [comment]
格莉丝别装了，我家邦布不会让你摸的

**Comment by XiaoFffeng** [comment]
格莉丝，我知道是你

**Comment by yuluono** [comment]
嗯呢嗯呢！
